dotnet-client-libraries/TestcontainersForDotNetAndDocker/TestcontainersForDotNetAndDocker.Tests/CatsApiFactory.cs

```diff
@@ -0,0 +1,58 @@
+﻿using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Containers;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using TestcontainersForDotNetAndDocker.Database;
+
+namespace TestcontainersForDotNetAndDocker.Tests;
+
+public class CatsApiFactory : WebApplicationFactory<Program>, IAsyncLifetime
```

Comment:
Can we maybe call this `CatsApiApplicationFactory` what do you think? Makes clearer what it is about.